### ISSUE TITLE
`.animateTo` not working

### ISSUE THREAD

**Issue by Matix-Media** [TesteurManiak/flutter_map_animations#42]
I do not know what I am doing wrong, but when calling `animatedMapController.animateTo(dest: dest)`, The camera location does not update..

My code to update the camera position:
```dart
      debugPrint("Now updating camera location");
      carController.animatedMapController.animateTo(
        dest: currentCheckpoint,
        duration: const Duration(milliseconds: 100),
      );
      debugPrint("Updated all stats");
```

My map widget:
```dart
                        FlutterMap(
                          mapController:
                              controller.animatedMapController.mapController,
                          options: const MapOptions(
                            initialZoom: 1,
                            interactionOptions: InteractionOptions(
                              flags: InteractiveFlag.none,
                            ),
                          ),
                          children: [
                            const ConfiguredTileLayer(),
                            const ConfiguredAttributionLayer(),
                            Obx(() {
                              return PolylineLayer(
                                polylines: [
                                  Polyline(
                                    points: (controller.ownActiveRide.value
                                            as OwnActiveTrackedRide)
                                        .checkpoints,
                                    color: controller.mapMarkingColor,
                                    strokeWidth: 5,
                                  ),
                                ],
                              );
                            }),
                          ],
                        ),
```

The log statement before the animateTo gets printed to console, but not the second one.

I get no errors or other stuff like that in the console so I don't really understand whats wrong.

**Comment by TesteurManiak** [comment]
Can you provide a complete reproducible example? Just from your code snippet it will be hard to help.

**Comment by Matix-Media** [comment]
yes, give me a minute

**Comment by Matix-Media** [comment]
Do you know of a good codesandbox for flutter?

**Comment by Matix-Media** [comment]
@TesteurManiak okay, so while creating the reproducable example I notice my use of `GetSingleTickerProviderStateMixin`, which caused issues due to multiple ticker providers being required. I did not notice my issue in my main project since I am executing the animateTo inside of `background_location` update callback. I also did not await the animateTo which caused the error to not be displayed in the console.

So my issue is now resolved and can be traced down to user error 😳
